### PR TITLE
Api play

### PR DESCRIPTION
**Pull request by tru36rit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 bundle.js
 configs
+.DS_Store
```
Comment (ghaynes1292) [inline .gitignore]: why is this here

File: src/components/App.js [hunk 2]
```diff
@@ -32,12 +32,26 @@ class App extends React.Component {
...
     let url = `http://www.omdbapi.com/?&apikey=${OMDB_KEY}&t=${title}`;
-    axios
-      .get(url)
-      .then(movie => this.setState({ movies: [movie.data, ...dummyMovieList] }))
-      .catch(err => err);
+    let movie = await axios.get(url);
+    this.setState({ movies: [movie.data, ...dummyMovieList] });
+  }
+
+  shuffleList(list) {
```
Comment (ghaynes1292) [inline src/components/App.js]: think you are mutating this. Bad idea

File: src/components/App.js [hunk 3]
```diff
@@ -32,12 +32,26 @@ class App extends React.Component {
...
+    let currentIndex = list.length,
+      temporaryValue,
```
Comment (lucas-aragno) [inline src/components/App.js]: it's a good practice to avoid this and use `let` everytime you declare a variable

File: src/components/App.js [hunk 4]
```diff
@@ -32,12 +32,26 @@ class App extends React.Component {
...
+      randomIndex;
+
+    while (0 !== currentIndex) {
```
Comment (lucas-aragno) [inline src/components/App.js]: I recommend this approach: https://gist.github.com/guilhermepontes/17ae0cc71fa2b13ea8c20c94c5c35dc4

Comment (ghaynes1292) [inline src/components/App.js]: Unsure if I like this shuffle algorithm. Maybe someone else has a stronger opinion than me

Comment (tru36rit) [reply src/components/App.js]: OMG so much better. Done.

File: .gitignore (hunk 1 above)
Comment (tru36rit) [reply .gitignore]: Not sure.  One of my commits was seeing that it was ignored.  I need to look more into how .DS_Store is working.

File: src/components/App.js (hunk 4 above)
Comment (tru36rit) [reply src/components/App.js]: Lucas said the same thing.  I replaced it.

File: src/components/App.js (hunk 2 above)
Comment (tru36rit) [reply src/components/App.js]: Aware of this one.  I don't mind in this case since it's shuffling for a really dumb reason which is because I was sick of seeing the same movies listed in the same order.

Still I'll make it copy and replace as opposed to mutate.

File: src/components/App.js (hunk 3 above)
Comment (tru36rit) [reply src/components/App.js]: Good to know thx.
So...
let currentIndex = list.length;
let temporaryValue;

Comment (tru36rit) [reply src/components/App.js]: Done (gone)

File: src/components/App.js (hunk 2 above)
Comment (tru36rit) [reply src/components/App.js]: Done (can you check I did it right?)

Comment (ghaynes1292) [reply src/components/App.js]: You are no longer mutating the parameter.
